docs/features/ulfm.rst

```diff
@@ -291,22 +261,94 @@ Open MPI level options
   latency (typically 1us increase).* You may want to **enable this
   option if you experience false positive** processes incorrectly
   reported as failed with the Open MPI failure detector.
+  This option is only relevant when ``mpi_ft_detector`` is ``true``.
 * ``mpi_ft_detector_period <float> (default: 3e0 seconds)`` heartbeat
   period. Recommended value is 1/3 of the timeout. _Values lower than
   100us may impart a noticeable effect on latency (typically a 3us
   increase)._
+  This option is only relevant when ``mpi_ft_detector`` is ``true``.
 * ``mpi_ft_detector_timeout <float> (default: 1e1 seconds)`` heartbeat
   timeout (i.e. failure detection speed). Recommended value is 3 times
   the heartbeat period.
+  This option is only relevant when ``mpi_ft_detector`` is ``true``.
 
 Known Limitations in ULFM
-^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------
 
 * InfiniBand support is provided through the UCT BTL; fault tolerant
   operation over the UCX PML is not yet supported for production runs.
 * TOPO, FILE, RMA are not fault tolerant. They are expected to work
   properly before the occurence of the first failure.
 
+Modified, Untested and Disabled Components
+------------------------------------------
+
+Frameworks and components are listed below and categorized into one of
+three classifications:
+
+1. **Modified:** This framework/component has been specifically modified
+                 such that it will continue to work after a failure.
+2. **Untested:** This framework/component has not been modified and/or
+                 tested with fault tolerance scenarios, and _may_ malfunction
+                 after a failure.
+3. **Disabled:** This framework/component will cause unspecified behavior when
+                 fault tolerance is enabled.
```

Comment:
```suggestion
1. **Modified:** This framework/component has been specifically modified
    such that it will continue to work after a failure.
2. **Untested:** This framework/component has not been modified and/or
   tested with fault tolerance scenarios, and _may_ malfunction
   after a failure.
3. **Disabled:** This framework/component will cause unspecified behavior when
    fault tolerance is enabled.
```

This doesn't render the bullets correctly with the additional spacing on the 2nd+3rd lines after the bullet start.